### ISSUE TITLE
Non-zero exit status on failure

### ISSUE THREAD

**Issue by adrian-gomez** [readmeio/rdme#407]
It would be great to have a non-zero exit status if a command fails (specifically `rdme openapi`) that way our ci will fail the build instead of reporting success when it wasn't successful.

Thanks for all the hard work!

**Comment by kanadgupta** [comment]
Hi @adrian-gomez! We do have some logic [here](https://github.com/readmeio/rdme/blob/0bc241ce5f5da88b28b949fef782854d58b55e18/bin/rdme#L26) that _**should**_ return a non-zero exit code whenever there is a failure. Do you have an example command that improperly returns a zero exit code? You can also email us at [support@readme.io](mailto:support@readme.io) if you have an OpenAPI file for us.

**Comment by adrian-gomez** [comment]
hi @kanadgupta, it seems to happen when network requests fail:
```
$ rdme openapi --id=ID --key=KEY public/swagger/built.json


 We're sorry, your upload request timed out. Please try again or split your file up into smaller chunks.


$ echo $?
0
```

**Comment by kanadgupta** [comment]
Hi @adrian-gomez! This has been fixed in `v6.1.3`. Thanks for calling this out!
